.travis.yml

```diff
@@ -71,5 +71,6 @@ jobs:
           if: tag IS present
           env: DMD=1.081.* F=production
           script:
+              - beaver dlang install
```

Comment:
I think regular `beaver install` should be sufficient here.

Reply:
Yup! Updated.